README.md

```diff
@@ -301,6 +301,9 @@ cd vcpkg
 
 If you previously installed VCPKG and dependencies, you may need to run `.\vcpkg.exe upgrade --no-dry-run` to upgrade to the latest packages.
 
+> Note: Setting up a development environment in windows without VCPKG is not supported. It requires installing all dev-dependencies globally and manually setting cmake files to link each of them.
+
+

```

Comment:
nit: Remove extra new line. If you use a markdown linter extension in VSCode, it would catch such changes.